files/es/web/api/fetch_api/using_fetch/index.html

```diff
@@ -348,24 +348,11 @@ <h2 id="Polyfill">Polyfill</h2>
 
 <h2 id="Especificaciones">Especificaciones</h2>
 
-<table class="standard-table">
- <tbody>
-  <tr>
-   <th scope="col">Especificación</th>
-   <th scope="col">Estado</th>
-   <th scope="col">Comentario</th>
-  </tr>
-  <tr>
-   <td>{{SpecName('Fetch')}}</td>
-   <td>{{Spec2('Fetch')}}</td>
-   <td>Definición inicial</td>
-  </tr>
- </tbody>
-</table>
+{{Specifications}}
 
 <h2 id="Compatibilidad_en_navegadores">Compatibilidad en navegadores</h2>
 
-<p>{{Compat("api.WindowOrWorkerGlobalScope.fetch")}}</p>
+{{Compat}}
```

Comment:
@yin1999 I may be wrong on this but if we are not using the compatibility’s name here, it should be placed in the metadata section at the beginning of the file, which means we cannot merge it until you add that `compat` metadata. Does that make sense to you?

Reply:
Because `mdn/yari` will merging metadata from `en-US`, when the `browser-compat` metadata does not exist in l10n documents, the metadata in `en-US` will be used for `{{Compat}}` macro. See: https://github.com/mdn/yari/blob/76d039511287583af9fb7ae89c2601fafd5f1862/kumascript/index.js#L58-L67

So when the `browser-compat` metadata is changed in `mdn/content`(such as: mdn/content#18328), we can just leave the related document as it is.

But if you guys think it's better to add `browser-compat` metadata, I'll do this.

Reply:
today I learned!